### ISSUE TITLE
Contributing.md

### ISSUE THREAD

**Issue by BTANISHA11** [Ghost-Ashu/Back-To-Code#48]
I want to add some section in it
How to make a pull request
How to create an issue
How to create a new branch
How to clone a repository 
Pls assign this to me @Ghost-Ashu

**Comment by Ghost-Ashu** [comment]
> 

It's easy though u will learn it  within no time

**Comment by BTANISHA11** [comment]
Yes so should I do this?
Its my first time doing contribution.

**Comment by Ghost-Ashu** [comment]
> Yes so should I do this? Its my first time doing contribution.

Yes ofcourse you should take part in it and happily contribute to open source. This is my Telegram @Ghost_Ashu
If any queries u can DM me
